doc/manuals/admin/crate.md

```diff
@@ -0,0 +1,9 @@
+# Crate
+
+[**Crate**](https://crate.io) is Quantumleap's default backend where NGSI data will be persisted. Until Quantumleap implements its querying API, you can interact directly with Crate to query all the data QL has saved from the received notifications.
+
+If you followed the [Installation Guide](./index.md), you have already Crate running in a Docker container, ready to be used.
+
+The easiest way to interact with it is using its admin interface, as documented [here](https://crate.io/docs/crate/guide/getting_started/connect/admin_ui.html). If not, you can use its [HTTP api](https://crate.io/docs/crate/guide/getting_started/connect/rest.html), or any of its supported drivers for different programming languages.
```

Comment:
The easiest way to interact with Crate is using its admin interface, as documented [here](https://crate.io/docs/crate/guide/getting_started/connect/admin_ui.html). Alternatively, you can use its [HTTP API](https://crate.io/docs/crate/guide/getting_started/connect/rest.html), or any of the [supported clients](https://crate.io/docs/crate/guide/getting_started/clients/index.html).